### PR TITLE
Add support for rundeck 3.x

### PR DESCRIPTION
**Pull request by smasa90**
#### Pull Request (PR) description
Quickwin to get rundeck 3.x running without error on:
- no more existant WEB-INF/web.xml 
- log4j.properties file location in rundeck-config.groovy

-> https://rundeck.org/news/2018/07/27/rundeck-3.0.0.html

### CODE REVIEW

**Comment by smasa90** [comment]
Don't know what mean these failing Travis checks :

**Comment by alexjfisher** [comment]
@smasa90 A new version of beaker-puppet has been released.  I restarted your build and the fix looks like it worked for this project. (There are other modules still experiencing issues due to other regressions).

**Comment by smasa90** [comment]
Big Thanks to you

File: spec/acceptance/rundeck_spec.rb [hunk 1]
```diff
@@ -7,9 +7,7 @@
       class { 'java':
         distribution => 'jre'
       }
-      class { 'rundeck':
-        package_ensure => '2.11.5'
-      }
+      class { 'rundeck': }
```
Comment (bastelfreak) [inline spec/acceptance/rundeck_spec.rb]: I think it would be nice to run ne acceptance test with 2.11.5 and afterwards check if an upgrade to 3.x works.

Comment (bastelfreak) [reply spec/acceptance/rundeck_spec.rb]: docs are, uhm, rare for beaker :(. it basically executes the puppet code in the spec/acceptance/rundeck_spec.rb file from top to bottom. There are already to blocks with rundeck 2.11.5. They will be executed in the same docker container. You can add another block at the bottom with the latest rundeck version, then puppet will upgrade from 2.11.5 to 3.

**Comment by bastelfreak** [comment]
Thanks!

**Comment by bastelfreak** [comment]
@alexjfisher I don't think so. Current users stay on their version 2.11.5. A new user will default to 3.

**Pull request merged**